src/components/node-list/node-list-group.js

```diff
@@ -32,7 +34,10 @@ export const NodeListGroup = ({
         'pipeline-nodelist__group--all-unset': allUnset,
       }
     )}>
-    <h3 className="pipeline-nodelist__heading">
+    <h3
+      className="pipeline-nodelist__heading"
+      onMouseEnter={() => onSectionMouseEnter(id)}
```

Comment:
Nice, I think even potentially move these mouse events up to the `pipeline-nodelist__group` element, so the whole section can be hovered